doc/content/reference/interop-repository/_index.md

```diff
@@ -7,46 +7,35 @@ description: ""
 
 <!-- more -->
 
-{{% tts %}} reads configuration from a file system. The root should contain `config.yml`, which contains various Join Servers.
+{{% tts %}} reads configuration from a file system. The root should contain `config.yml`, which contains Join Servers.
 
 ```yml
-join-servers:                 # list of Join Server interoperability configurations,
-                              # used to map a JoinEUI to the Join Server
+join-servers:
   - file: './path/js.yml'     # relative path to a file containing Join Server configuration
+    components: ['as', 'ns']  # optional selector for components (new in 3.21.0)
     join-euis:                # list of Join EUI prefixes the Join Server should handle
-    - '11aa000000000000/16'
+    - '11AA000000000000/16'   # in this example, the first 16 bits, so all JoinEUIs starting with 11AA
 ```
 
-All paths are relative to the file they are defined in, that is `example/js.yml` defined in `interopconf/config.yml` is expected to be located at `interopconf/example/js.yml`.
-
-In case `JoinEUI` prefixes overlap, the most specific prefix takes precedence.
+All paths are relative to the `config.yml` file they are defined in.
 
-{{% tts %}} Network Server always first tries the cluster-local Join Server and if it's not found or the device is not found, a Join Server via this interoperability is contacted.
+{{% tts %}} Network Server always first tries the cluster-local Join Server. If the cluster-local Join Server does not accept the join-request, a Join Server via the LoRaWAN Backend Interfaces interoperability is contacted.
```

Comment:
`If the cluster-local Join Server does not accept the join-request, a Join Server via the LoRaWAN Backend Interfaces interoperability is contacted.` things sounds a bit too much like Yoda to me. 

Maybe `If the cluster-local Join Server does not accept the join-request, a Join Server is contacted via the LoRaWAN Backend Interfaces interoperability.` ?